update.ps1

```diff
@@ -232,29 +196,26 @@ try {
 
                 $body = ($updateAccountObject | ConvertTo-Json -Depth 10)
                 $splatWebRequest = @{
-                    Uri             = "$($c.BaseUri)/medewerkers/$($currentAccount.Id)"
+                    Uri             = $uri
                     Headers         = $headers
                     Method          = "PUT"
                     Body            = ([System.Text.Encoding]::UTF8.GetBytes($body))
-                    ContentType     = "application/json;charset=utf-8"
+                    #ContentType     = "application/json;charset=utf-8"
                     UseBasicParsing = $true
                 }
 
                 Write-Verbose "Updating account [$($currentAccount.RoepNaam) $($currentAccount.AchterNaam) ($($currentAccount.Id))]. Account object: $($updateAccountObject | ConvertTo-Json -Depth 10)"
 
-                if (-not($dryRun -eq $true)) {
+                if (-not($actionContext.DryRun -eq $true)) {
                     $updatedAccount = Invoke-RestMethod @splatWebRequest -Verbose:$false
 
-                    # Set aRef object for use in futher actions
-                    $aRef = [PSCustomObject]@{
-                        Id = $updatedAccount.Id
-                    }
-
-                    $auditLogs.Add([PSCustomObject]@{
-                            # Action  = "" # Optional
+                    $outputContext.success = $true
+                    $outputContext.AuditLogs.Add([PSCustomObject]@{
                             Message = "Successfully updated account [$($currentAccount.RoepNaam) $($currentAccount.AchterNaam) ($($updatedAccount.Id))]. Updated properties: $($changedPropertiesObject | ConvertTo-Json -Depth 10)"
+                            #Message = 'Update account was successful'
```

Comment:
I assume this is also used for testing. If that’s the case, please clean this up. If not, adjust the code to use the correct message.

Reply:
removed second message